assets/xml/static/nintendo_rogo_static.xml

```diff
@@ -0,0 +1,8 @@
+<Root>
+    <File Name="nintendo_rogo_static" Segment="1" Game="MM">
```

Comment:
```suggestion
    <File Name="nintendo_logo_static" Segment="1" Game="MM">
```
I'm guessing `rogo` was original? Can we change it?

Reply:
That's the filename, same as OoT. And no, we can't change it only here or ZAPD won't be able to find the file.